src/Core/src/Platform/iOS/ApplicationExtensions.cs

```diff
@@ -66,6 +66,14 @@ public static void CreatePlatformWindow(this IUIWindowSceneDelegate sceneDelegat
 			{
 				sceneDelegate.SetWindow(window);
 				sceneDelegate.GetWindow()?.MakeKeyAndVisible();
+
+				UIScreen.Notifications.ObserveModeDidChange(window.Screen, new System.EventHandler<NSNotificationEventArgs>((sender, args) =>
```

Comment:
We need to sort this out still and somehow capture the observer token returned... Not sure how to best do this as we aren't subclassing UIWindow...

Reply:
Seems the notification do not fire on mac catalyst (but they do on iOS) 

Fix later #5315